waypoints_x: [-5.0, -27.0, -25.0, 1.0]
waypoints_y: [-5.0, 4.0, -21.0, -21.0]
waypoint_tolerance: 2.0
